dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -11,6 +11,19 @@ function createVideoWithFallback(parentElement, options, width, height) {
   return video;
 }
 
+function onVideoEnded() {
+  $('.video-modal').trigger("ended");
+}
+
+function onYouTubeIframeAPIReady() {
+  var player = YT.get('video') || new YT.Player('video');
+  player.addEventListener('onStateChange', function (state) {
+    if (state.data === YT.PlayerState.ENDED) {
+      onVideoEnded();
+    }
+  });
+}
+
```

Comment:
will `onYouTubeIframeAPIReady` get called separately for each individual video on the page, or are we making the assumption that we'll only ever load a single video on a page?